### PR TITLE
Chapter 6 Author Review

### PR DESCRIPTION
**Pull request by szimmer**

### CODE REVIEW

File: 06-statistical-testing.Rmd [hunk 1]
```diff
@@ -8,7 +10,7 @@ When analyzing results from a survey, the point estimates described in Chapter \
...
+Up to this point, the functions we've provided have used the wrappers from the {srvyr} package. This means that the functions work with tidyverse syntax. However, the functions in this chapter do not have wrappers from the {srvyr} package and are instead used directly from the {survey} package. This means that the design object is *not* the first argument and that to use these functions with the magrittr pipe `%>%` and tidyverse syntax, we will need to use dot (`.`) notation. Functions that work with the magrittr pipe `%>%` have the first argument as data. When data is run, it automatically places anything to the left of the pipe into the first argument of the function to the right of the pipe. For example, if we wanted to take the `mtcars` data and filter to only cars with six cylinders, we can write the code in one of three ways:
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:13]:
```suggestion
Up to this point, the functions we've provided have used the wrappers from the {srvyr} package. This means that the functions work with tidyverse syntax. However, the functions in this chapter do not have wrappers from the {srvyr} package and are instead used directly from the {survey} package. This means that the design object is *not* the first argument and that to use these functions with the magrittr pipe `%>%` and tidyverse syntax, we will need to use dot (`.`) notation. Functions that work with the magrittr pipe `%>%` have the first argument as data. When data is run, it automatically places anything to the left of the pipe into the first argument of the function to the right of the pipe. For example, if we wanted to take the `mtcars` data and filter to only cars with six cylinders, we can write the code in at least four different ways:
```

File: 06-statistical-testing.Rmd [hunk 2]
```diff
@@ -89,7 +91,7 @@ recs_des <- recs_in %>%
 
 ## Comparison of Proportions and Means {#stattest-ttest}
 
-To compare two proportions or means, we use t-tests. This allows us to determine if one proportion or mean is statistically different from the other. T-tests are commonly used to determine if a single estimate differs from a known value (e.g., 0 or 50%) or to compare two group means (e.g., males vs. females). Comparing a single estimate to a known value is called a *one sample t-test*, and we can set up the hypothesis test as follows:
+To compare two proportions or means, we use t-tests. This allows us to determine if one proportion or mean is statistically different from the other. T-tests are commonly used to determine if a single estimate differs from a known value (e.g., 0 or 50%) or to compare two group means (e.g., males vs. females). Comparing a single estimate to a known value is called a *one sample t-test*, and we can set up the hypothesis test as follows:
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:94]:
```suggestion
To compare two proportions or means, we use t-tests. This allows us to determine if one proportion or mean is statistically different from the other. t-tests are commonly used to determine if a single estimate differs from a known value (e.g., 0 or 50%) or to compare two group means (e.g., males vs. females). Comparing a single estimate to a known value is called a *one sample t-test*, and we can set up the hypothesis test as follows:  
```

File: 06-statistical-testing.Rmd [hunk 3]
```diff
@@ -124,14 +126,14 @@ Notice that the first argument here is the `formula` and not the `design`. This
...
+2. **Two-sample t-test:**  
+  a. **Unpaired:**  
+    - **2 level grouping variable:** `var ~ groupVar`, where `var` is the measure of interest and `groupVar` is a variable with two categories. For example, we could test if the proportion of the population that has blue eyes is different for children aged 5-10 years old compared to children under 5 years old.   
+    - **3+ level grouping variable:** `var ~ I(groupVar == level)`, where `var` is the measure of interest, `groupVar` is the categorical variable, and `level` is the category level to isolate. Again we need to use the `I()` function to tell the program to isolate the category before doing the comparison across groups. For example, we could test if the test scores in one classroom differed from all other classrooms.   
+  b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:136]: Indentation matters

```suggestion
1. **One-sample t-test:** 
    a. **Comparison to 0:** `var ~ 0`, where `var` is the measure of interest, and `0` is the value we compare it to. For example, we could test if the proportion of the population that has blue eyes is different from `0`.
    b. **Comparison to a different value:** `I(var - value) ~ 0`, where `var` is the measure of interest and `value` is what we are comparing to. We need to use the `I()` function to tell the program to calculate the difference between the variable and the comparison value before testing. For example, we could test if the proportion of the population that has blue eyes is different from `25%` by using `I(var - 0.25)~0`.
2. **Two-sample t-test:**
    a. **Unpaired:**
        - **2 level grouping variable:** `var ~ groupVar`, where `var` is the measure of interest and `groupVar` is a variable with two categories. For example, we could test if the proportion of the population that has blue eyes is different for children aged 5-10 years old compared to children under 5 years old. 
        - **3+ level grouping variable:** `var ~ I(groupVar == level)`, where `var` is the measure of interest, `groupVar` is the categorical variable, and `level` is the category level to isolate. Again we need to use the `I()` function to tell the program to isolate the category before doing the comparison across groups. For example, we could test if the test scores in one classroom differed from all other classrooms.
    b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.
```

File: 06-statistical-testing.Rmd [hunk 4]
```diff
@@ -148,12 +150,12 @@ ANES asked respondents if they voted for president in the 2020 election. In our
 
 voteprop <- anes_des %>%
  group_by(VotedPres2020) %>%
- summarize(survey_prop())
+ summarize(survey_prop())
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:153]:
```suggestion
 summarize(p=survey_prop())  
```

File: 06-statistical-testing.Rmd [hunk 5]
```diff
@@ -148,12 +150,12 @@ ANES asked respondents if they voted for president in the 2020 election. In our
...
 voteprop <- anes_des %>%
  group_by(VotedPres2020) %>%
- summarize(survey_prop())
+ summarize(survey_prop())  
 
 voteprop
 ```
 
-Based on this, `r signif((voteprop %>% filter(VotedPres2020=="Yes") %>% pull(coef))*100,3)`% of the U.S. voting-eligible population voted for president in 2020. If we wanted to know how this compares to other countries, we could use `svyttest()`. For example, if we know that the voter turnout in Germany in the 2017 general election was 76.2%, we could set up our hypothesis as follows:
+Based on this, `r signif((voteprop %>% filter(VotedPres2020=="Yes") %>% pull(coef))*100,3)`% of the U.S. voting-eligible population voted for president in 2020. If we wanted to know how this compares to other countries, we could use `svyttest()`. For example, if we know that the voter turnout in Germany in the 2017 general election was 76.2%, we could set up our hypothesis as follows:
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:158]:
```suggestion
Based on this, `r signif((voteprop %>% filter(VotedPres2020=="Yes") %>% pull(p))*100,3)`% of the U.S. voting-eligible population voted for president in 2020. If we wanted to know how this compares to other countries, we could use `svyttest()`. For example, if we know that the voter turnout in Germany in the 2017 general election was 76.2%, we could set up our hypothesis as follows:  
```

File: 06-statistical-testing.Rmd [hunk 6]
```diff
@@ -216,7 +218,7 @@ Or, we could take our t-test estimate (`ttest_ex2$estimate`) and add it to 68:
 ttest_ex2$estimate + 68
 ```
 
-The result is the same in both methods, so we see that the average temperature U.S. Households set their thermostat to in the summer at night is `r signif(ttest_ex2$estimate + 68,3)`. Looking at the output from the `svyttest()`, the t-statistic is `r ttest_ex2$statistic`, and the p-value is `r ttest_ex2$p.value`, indicating that the average is statistically different from 68$^\circ$F at an $\alpha$ level of $0.05$.
+The result is the same in both methods, so we see that the average temperature U.S. Households set their thermostat to in the summer at night is `r signif(ttest_ex2$estimate + 68,3)`. Looking at the output from the `svyttest()`, the t-statistic is `r ttest_ex2$statistic`, and the p-value is `r ttest_ex2$p.value`, indicating that the average is statistically different from 68$^\circ$F at an $\alpha$ level of $0.05$.
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:221]:
```suggestion
The result is the same in both methods, so we see that the average temperature U.S. Households set their thermostat to in the summer at night is `r signif(ttest_ex2$estimate + 68,3)`. Looking at the output from the `svyttest()`, the t-statistic is `r signif(ttest_ex2$statistic, 3)`, and the p-value is `r signif(ttest_ex2$p.value, 3)`, indicating that the average is statistically different from 68$^\circ$F at an $\alpha$ level of $0.05$.  
```

File: 06-statistical-testing.Rmd [hunk 7]
```diff
@@ -246,7 +248,7 @@ ttest_ex3 <- recs_des %>%
 ttest_ex3
 ```
 
-The results indicate that the difference in electrical bills for those that used AC and those that did not is, on average, \$`r round(ttest_ex3$estimate,2)`. The difference appears to be statistically significant as the t-statistic is `r ttest_ex3$statistic` and the p-value is `r ttest_ex3$p.value`.
+The results indicate that the difference in electrical bills for those that used AC and those that did not is, on average, \$`r round(ttest_ex3$estimate,2)`. The difference appears to be statistically significant as the t-statistic is `r ttest_ex3$statistic` and the p-value is `r ttest_ex3$p.value`.
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:251]:
```suggestion
The results indicate that the difference in electrical bills for those that used AC and those that did not is, on average, \$`r round(ttest_ex3$estimate,2)`. The difference appears to be statistically significant as the t-statistic is `r ttest_ex3$statistic` and the p-value is `r signif(ttest_ex3$p.value, 3)`.   
```

File: 06-statistical-testing.Rmd [hunk 8]
```diff
@@ -268,7 +270,7 @@ ttest_ex4 <- recs_des %>%
 ttest_ex4
 ```
 
-U.S. households set their thermostat on average `r signif(ttest_ex4$estimate,2)`$^\circ$F warmer in summer nights than winter nights, which is statistically significant (t = `r ttest_ex4$statistic`, p-value = `r ttest_ex4$p.value`).
+U.S. households set their thermostat on average `r signif(ttest_ex4$estimate,2)`$^\circ$F warmer in summer nights than winter nights, which is statistically significant (t = `r ttest_ex4$statistic`, p-value = `r ttest_ex4$p.value`).
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:273]:
```suggestion
U.S. households set their thermostat on average `r signif(ttest_ex4$estimate,2)`$^\circ$F warmer in summer nights than winter nights, which is statistically significant (t = `r signif(ttest_ex4$statistic, 3)`, p-value = `r signif(ttest_ex4$p.value, 3)`).  
```

File: 06-statistical-testing.Rmd [hunk 9]
```diff
@@ -427,8 +429,8 @@ chi_ex1 <- anes_des_educ %>%
 chi_ex1
 ```
 
-The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; $p-value=$ `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:
-<!--This doesn't render correctly in HTML but does ok in PDF.-->
+The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; $p-value=$ `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:432]:
```suggestion
The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; p-value= `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:  
```

File: 06-statistical-testing.Rmd [hunk 10]
```diff
@@ -427,8 +429,8 @@ chi_ex1 <- anes_des_educ %>%
 chi_ex1
 ```
 
-The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; $p-value=$ `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:
-<!--This doesn't render correctly in HTML but does ok in PDF.-->
+The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; $p-value=$ `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:  
+<!--This doesn't render correctly in HTML but does ok in PDF.-->
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:433]: Not sure what you don't think looks good in HTML. Share details/screenshot in Google doc or email

File: 06-statistical-testing.Rmd [hunk 11]
```diff
@@ -473,8 +475,8 @@ ex1_table %>%
   ) %>%
   ggplot(aes(x = Education, y = Proportion, color = Names)) +
   geom_point(alpha = 0.5) +
-  geom_errorbar(aes(ymin = Observed_low, ymax = Observed_upp)) +
-  theme_bw()
+  geom_errorbar(aes(ymin = Observed_low, ymax = Observed_upp)) +   
+  theme_bw()
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:479]:
```suggestion
  theme_bw() +
  scale_color_manual(name = "Type", values = book_colors[c(4,1)]) +
  theme(legend.position = "bottom")
```

File: 06-statistical-testing.Rmd [hunk 12]
```diff
@@ -553,13 +555,13 @@ We may find it easier to look at these proportions graphically. We can use `gglo
...
-  geom_errorbar(aes(ymin = Observed_low, ymax = Observed_upp)) +
-  ylab("Proportion") +
-  theme_bw()
+chi_ex2_obs %>%  
+  ggplot(aes(x = TrustGovernment, y = Observed, color = TrustGovernment)) +  
+  facet_wrap( ~ TrustPeople, ncol = 5) +  
+  geom_point() +  
+  geom_errorbar(aes(ymin = Observed_low, ymax = Observed_upp)) +    
+  ylab("Proportion") +      
+  theme_bw()
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:564]:
```suggestion
chi_ex2_obs %>%
  mutate(TrustPeople=str_c("Trust in People:\n", TrustPeople)) %>%
  ggplot(aes(x = TrustGovernment, y = Observed, color = TrustGovernment)) +
  facet_wrap( ~ TrustPeople, ncol = 5, labeller = ) +
  geom_point() +
  geom_errorbar(aes(ymin = Observed_low, ymax = Observed_upp)) +
  ylab("Proportion") +
  xlab("") +
  theme_bw() +
  scale_color_manual(name="Trust in Government", values=book_colors) +
  theme(axis.text.x = element_blank(), 
        axis.ticks.x = element_blank(),
        legend.position = "bottom")
```

File: 06-statistical-testing.Rmd [hunk 13]
```diff
@@ -569,12 +571,12 @@ Researchers and politicians often look at specific demographics each election cy
...
-    p_{3_{Biden}} = p_{3_{Trump}} = p_{3_{Other}},\\
-    p_{4_{Biden}} = p_{4_{Trump}} = p_{4_{Other}},\\
-    p_{5_{Biden}} = p_{5_{Trump}} = p_{5_{Other}},\\
-    p_{6_{Biden}} = p_{6_{Trump}} = p_{6_{Other}}$
+- $H_0: p_{1_{Biden}} = p_{1_{Trump}} = p_{1_{Other}},\\  
+    p_{2_{Biden}} = p_{2_{Trump}} = p_{2_{Other}},\\  
+    p_{3_{Biden}} = p_{3_{Trump}} = p_{3_{Other}},\\  
+    p_{4_{Biden}} = p_{4_{Trump}} = p_{4_{Other}},\\  
+    p_{5_{Biden}} = p_{5_{Trump}} = p_{5_{Other}},\\  
+    p_{6_{Biden}} = p_{6_{Trump}} = p_{6_{Other}}$
```
Comment (szimmer) [inline 06-statistical-testing.Rmd:579]:
```suggestion
- \begin{align*}
H_0: p_{1_{Biden}} &= p_{1_{Trump}} = p_{1_{Other}},\\
    p_{2_{Biden}} &= p_{2_{Trump}} = p_{2_{Other}},\\
    p_{3_{Biden}} &= p_{3_{Trump}} = p_{3_{Other}},\\
    p_{4_{Biden}} &= p_{4_{Trump}} = p_{4_{Other}},\\
    p_{5_{Biden}} &= p_{5_{Trump}} = p_{5_{Other}},\\
    p_{6_{Biden}} &= p_{6_{Trump}} = p_{6_{Other}}
  \end{align*}
```

File: 06-statistical-testing.Rmd [hunk 14]
```diff
@@ -1,5 +1,7 @@
 # Statistical testing {#c06-statistical-testing}
 
+<!-- author review -->
+
 When analyzing results from a survey, the point estimates described in Chapter \@ref(c05-descriptive-analysis) help us understand the data at a high level. Still, researchers and the public often want to make comparisons between different groups. These comparisons are calculated through statistical testing, where we compare the point estimates and variance estimates of each statistic to see if statistically significant differences exist. The general idea of statistical testing is the same for data obtained through surveys and data obtained through other methods, but the importance lies in ensuring the variance is calculated correctly. Functions in the {survey} packages allow for the correct estimation of the variances. This chapter will cover the following statistical tests with survey data and functions:
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:5]: I call this "point estimates" in this chapter, but we only refer to them as descriptive statistics in chapter 5.  Thoughts?  Will need to either change it here, or add a reference to point estimates in chapter 5.

File: 06-statistical-testing.Rmd (hunk 1 above)
Comment (rpowell22) [inline 06-statistical-testing.Rmd:13]:
```suggestion
Up to this point, the functions we've provided have used the wrappers from the {srvyr} package. This means that the functions work with tidyverse syntax. However, the functions in this chapter do not have wrappers in the {srvyr} package and are instead used directly from the {survey} package. This means that the design object is *not* the first argument and that to use these functions with the magrittr pipe `%>%` and tidyverse syntax, we will need to use dot (`.`) notation. Functions that work with the magrittr pipe `%>%` have the first argument as data. When data is run, it automatically places anything to the left of the pipe into the first argument of the function to the right of the pipe. For example, if we wanted to take the `mtcars` data and filter to only cars with six cylinders, we can write the code in one of three ways:  
```

File: 06-statistical-testing.Rmd (hunk 6 above)
Comment (rpowell22) [inline 06-statistical-testing.Rmd:221]:
```suggestion
The result is the same in both methods, so we see that the average temperature U.S. Households set their thermostat to in the summer at night is `r signif(ttest_ex2$estimate + 68,3)`. Looking at the output from `svyttest()`, the t-statistic is `r ttest_ex2$statistic`, and the p-value is `r ttest_ex2$p.value`, indicating that the average is statistically different from 68$^\circ$F at an $\alpha$ level of $0.05$.  
```

File: 06-statistical-testing.Rmd [hunk 15]
```diff
@@ -246,7 +248,7 @@ ttest_ex3 <- recs_des %>%
 ttest_ex3
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:248]:
```suggestion
tidy(ttest_ex3)
```

File: 06-statistical-testing.Rmd (hunk 7 above)
Comment (rpowell22) [inline 06-statistical-testing.Rmd:251]:
```suggestion
The results indicate that the difference in electrical bills for those that used AC and those that did not is, on average, \$`r round(ttest_ex3$estimate,2)`. The difference appears to be statistically significant as the t-statistic is `r ttest_ex3$statistic` and the p-value is `r ttest_ex3$p.value`.   This means that households that used AC spent on average $462.90 more each [TIMEFRAME].
```

Comment (rpowell22) [reply 06-statistical-testing.Rmd:251]: Need to check what the timeframe is for this.

File: 06-statistical-testing.Rmd [hunk 16]
```diff
@@ -268,7 +270,7 @@ ttest_ex4 <- recs_des %>%
 ttest_ex4
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:270]:
```suggestion
tidy(ttest_ex4)
```

File: 06-statistical-testing.Rmd (hunk 11 above)
Comment (rpowell22) [reply 06-statistical-testing.Rmd:479]: Thank you!!

File: 06-statistical-testing.Rmd [hunk 17]
```diff
@@ -124,21 +126,21 @@ Notice that the first argument here is the `formula` and not the `design`. This
...
+  b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.  
 
 Additionally, the `na.rm` argument defaults to `FALSE`, which means if any data is missing, the t-test will not compute. Throughout this chapter we will always set `na.rm = TRUE`, but before analyzing the survey data, review the notes provided in Chapter \@ref(c04-understanding-survey-data-documentation) to better understand how to handle missing data. Finally, the `level` argument is $1-\alpha$, or the amount of type 1 error. The default is $0.95$.
 
 ### Examples {#stattest-ttest-examples}
 Let's walk through a few examples using the ANES and RECS data. See Section \@ref(stattest-setup) above to set up the design objects.
 
-#### Example 1: One-sample t-test {.unnumbered #stattest-ttest-ex1}
+#### Example 1: One-sample t-test {.unnumbered #stattest-ttest-ex1}
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:143]: I think we should flip ex1 and ex2.  ex1 is more complicated.

File: 06-statistical-testing.Rmd [hunk 18]
```diff
@@ -371,9 +373,9 @@ svychisq(
...
 
-For tests of independence, the `Wald` and `adjWald` are recommended as they provide a better adjustment for variable comparisons. If the data has a small number of primary sampling units (PSUs) compared to the degrees of freedom, then the `adjWald` statistic should be used to account for this. The `lincom` and `saddlepoint` statistics are available for more complicated data structures.<!--I'm struggling to find a lot of information around these last two statistics, any thoughts/suggestions?-->
+For tests of independence, the `Wald` and `adjWald` are recommended as they provide a better adjustment for variable comparisons. If the data has a small number of primary sampling units (PSUs) compared to the degrees of freedom, then the `adjWald` statistic should be used to account for this. The `lincom` and `saddlepoint` statistics are available for more complicated data structures.<!--I'm struggling to find a lot of information around these last two statistics, any thoughts/suggestions?-->
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:376]: Cite Lumley? Check to see how much his book goes into this detail.

File: 06-statistical-testing.Rmd [hunk 19]
```diff
@@ -523,10 +525,10 @@ chi_ex2$observed %>% as_tibble() %>%
   gt(rowname_col = "TrustGovernment") %>%
   tab_stubhead(label = "Trust in Government") %>%
   tab_spanner(label = "Trust in People",
-              columns = everything())
+              columns = everything())
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:528]: Table goes off page.  Need to adjust.  Think can just adjust the column headers but will need to play with it a bit.

File: 06-statistical-testing.Rmd [hunk 20]
```diff
@@ -523,10 +525,10 @@ chi_ex2$observed %>% as_tibble() %>%
   gt(rowname_col = "TrustGovernment") %>%
   tab_stubhead(label = "Trust in Government") %>%
   tab_spanner(label = "Trust in People",
-              columns = everything())
+              columns = everything()) 
 ```
 
-The second option is to use `group_by()` and `survey_mean()` functions to calculate the proportions from the ANES design object. A reminder that with more than one variable listed in the `group_by()` statement, the proportions are within the first variable listed. As mentioned above, we are looking at the distribution of `TrustGovernment` for each level of `TrustPeople`.
+The second option is to use `group_by()` and `survey_mean()` functions to calculate the proportions from the ANES design object. A reminder that with more than one variable listed in the `group_by()` statement, the proportions are within the first variable listed. As mentioned above, we are looking at the distribution of `TrustGovernment` for each level of `TrustPeople`.
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:531]: Will add reference to where we talk about this once we fix the ordering of chapter 5.

File: 06-statistical-testing.Rmd [hunk 21]
```diff
@@ -543,7 +545,7 @@ chi_ex2_obs %>%
   gt(rowname_col = "TrustGovernment") %>%
   tab_stubhead(label = "Trust in Government") %>%
   tab_spanner(label = "Trust in People",
-              columns = everything())
+              columns = everything())
```
Comment (rpowell22) [inline 06-statistical-testing.Rmd:548]: Output far off page.  May need to do landscape if we want to keep it in this format.  Also should adjust it so 3 decimals across all items.

**Review by rpowell22 — CHANGES_REQUESTED** [review]

File: 06-statistical-testing.Rmd (hunk 14 above)
Comment (ivelasq) [inline 06-statistical-testing.Rmd:5]: "The general idea of statistical testing is the same for data obtained through surveys and data obtained through other methods" - should we expand on what the general idea is? In particular, the goals and objectives of statistical testing.

File: 06-statistical-testing.Rmd [hunk 22]
```diff
@@ -120,25 +122,25 @@ svyttest(formula,
...
+  b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.  
 
 Additionally, the `na.rm` argument defaults to `FALSE`, which means if any data is missing, the t-test will not compute. Throughout this chapter we will always set `na.rm = TRUE`, but before analyzing the survey data, review the notes provided in Chapter \@ref(c04-understanding-survey-data-documentation) to better understand how to handle missing data. Finally, the `level` argument is $1-\alpha$, or the amount of type 1 error. The default is $0.95$.
```
Comment (ivelasq) [inline 06-statistical-testing.Rmd:138]: Is this `level` different than the one above? "3+ level grouping variable: var ~ I(groupVar == level), where var is the measure of interest, groupVar is the categorical variable, and level is the category level to isolate"

File: 06-statistical-testing.Rmd [hunk 23]
```diff
@@ -162,9 +164,9 @@ To conduct this in R, we would then use the `svyttest()` function:
 ```{r}
 #| label: stattest-ttest-ex1
 ttest_ex1 <- anes_des %>%
- svyttest(formula = I(I(VotedPres2020 == "Yes") - 0.762) ~ 0,
-      design = .,
-      na.rm = TRUE)
+  svyttest(formula = I(I(VotedPres2020 == "Yes") - 0.762) ~ 0,
```
Comment (ivelasq) [inline 06-statistical-testing.Rmd:167]: Maybe worth mentioning how you can compound `I()`

File: 06-statistical-testing.Rmd [hunk 24]
```diff
@@ -120,25 +122,25 @@ svyttest(formula,
...
-  b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.
+1. **One-sample t-test:**     
+  a. **Comparison to 0:** `var ~ 0`, where `var` is the measure of interest, and `0` is the value we compare it to. For example, we could test if the proportion of the population that has blue eyes is different from `0`.  
+  b. **Comparison to a different value:** `I(var - value) ~ 0`, where `var` is the measure of interest and `value` is what we are comparing to. We need to use the `I()` function to tell the program to calculate the difference between the variable and the comparison value before testing. For example, we could test if the proportion of the population that has blue eyes is different from `25%` by using `I(var - 0.25)~0`.
```
Comment (ivelasq) [inline 06-statistical-testing.Rmd:131]: Below, we use a formula with double `I()`. Maybe it's worth explaining what this is and when/how to use it early in the chapter.

File: 06-statistical-testing.Rmd [hunk 25]
```diff
@@ -182,9 +184,9 @@ The estimate that is presented here is actually the difference between the U.S.
 
 #### Example 2: One-sample t-test {.unnumbered #stattest-ttest-ex2}
 
-RECS asks respondents to indicate what temperature they set their house to during the summer at night. In our data, we've called this variable `SummerTempNight`. If we want to see if the U.S. household sets their temperature at a value different from 68$^\circ$F, we could set up the hypothesis as follows:
+RECS asks respondents to indicate what temperature they set their house to during the summer at night. In our data, we've called this variable `SummerTempNight`. If we want to see if the U.S. household sets their temperature at a value different from 68$^\circ$F, we could set up the hypothesis as follows:
```
Comment (ivelasq) [inline 06-statistical-testing.Rmd:187]: their average temperature?

File: 06-statistical-testing.Rmd (hunk 14 above)
Comment (ivelasq) [reply 06-statistical-testing.Rmd:5]: @szimmer since you are reviewing chapter 5, can we add 'point estimates' and a definition there, please?

**Pull request merged**